src/bitmap/store.rs

```diff
@@ -43,40 +45,72 @@ impl Store {
         }
     }
 
-    pub fn insert_range(&mut self, range: Range<u16>) -> u64 {
-        // A Range is defined as being of size 0 if start >= end.
-        if range.is_empty() {
+    pub fn insert_range<R>(&mut self, range: R) -> u64
+    where
+        R: RangeBounds<u16>,
+    {
```

Comment:
As we are in the `Store` we should accept a `RangeInclusive<u16>` here, that would simplify the usage of the library. Because we know that we are in a finished range of numbers i.e. `0..=u32::MAX`.